blockchain/chain_sync/src/sync.rs

```diff
@@ -699,9 +699,11 @@ where
             &self.chain_store.tipset_from_keys(header.parents())?,
         )?;
 
-        header
-            .validate_block_drand(Arc::clone(&self.beacon), prev_beacon)
-            .await?;
+        if std::env::var("LOTUS_IGNORE_DRAND") == Ok("1".to_owned()) {
```

Comment:
This probably doesn't need to be `LOTUS_`.. Can just trim that or have it prefixed with forest